infra.bs

```diff
@@ -1292,9 +1290,20 @@ when used with <a for=map>exists</a>.
 <hr>
 
 <p>To <dfn export for=map lt="get|get the value">get the value of an entry</dfn> in an
-<a>ordered map</a> given a <a for=map>key</a>, return the <a for=map>value</a> of the
-<a for=map>entry</a> whose <a for=map>key</a> is the given key. We can also use the indexing syntax
-explained above.
+<a>ordered map</a> <var>map</var> given a <a for=map>key</a> <var>key</var>:
+
+<ol>
+ <li><p>Assert: <var>map</var> <a for=map>contains</a> <var>key</var>.
+
+ <li><p>Return the <a for=map>value</a> of the <a for=map>entry</a> in <var>map</var> whose
+ <a for=map>key</a> is <var>key</var>.
+</ol>
+
+<p>We can also denote <a for=map lt=get>get the value of an entry</a> using an indexing syntax by
```

Comment:
"denote get" is weird; "denote getting" would work. Or alternate phrasings, e.g. "We can also express getting..."